channel-jira-server/src/main/java/com/synopsys/integration/alert/channel/jira/server/model/JiraServerGlobalConfigModel.java

```diff
@@ -25,15 +25,15 @@ public JiraServerGlobalConfigModel() {
         // For serialization
     }
 
-    public JiraServerGlobalConfigModel(String id, String name, String url, String userName) {
+    public JiraServerGlobalConfigModel(String id, String name, String url, String userName, String password) {
```

Comment:
If we are passing the password this way, would we want it to be @Nullable since the UI would return a password with the value "null"?

Reply:
We return null today for the email password field.  we have the same constraint and concern with the email channel global configuration.